src/promptflow/promptflow/_core/tool_meta_generator.py

```diff
@@ -239,6 +239,8 @@ def generate_tool_meta_dict_by_file(path: str, tool_type: ToolType):
         return generate_prompt_meta_dict(name, content, source=path)
     elif tool_type == ToolType.PROMPT:
         return generate_prompt_meta_dict(name, content, prompt_only=True, source=path)
+    elif tool_type == ToolType.CUSTOM_LLM:
```

Comment:
not sure if we also need to include executor guys to aware this change

Reply:
This type has already been added in constant class and I haven't change existing logic, so I think it's safe and just mean that we have no test on this before hh.

cc @thy09 for this. CustomLLM is not supported in tools meta generation before, any concern on updating this?

Reply:
approved and leave this comment open, once we confirm with Heyi, I think we can merge this.

Reply:
as discussed, I have generate `package_with_prompt` as `prompt` for now. will raise new pull request if we need to remove `type` and `source` in `flow.tools.json`